static/skin/viewer.js

```diff
@@ -204,14 +204,29 @@ function handle_visual_viewport_change() {
   contentIframe.height = wh - contentIframe.offsetTop - 4;
 }
 
+function setIframeUrl(path) {
+  try {
+    // Don't do anything if we are already at the requested URL.
+    // This is needed to break the infinite ping-pong played by
+    // handle_location_hash_change() and handle_content_url_change()
+    // (when either top-window or content window/iframe URL changes the other
+    // one is updated too).
+    if ( path == contentIframe.contentWindow.location.pathname )
+      return;
+  } catch ( error ) {
+    // This happens in Firefox when a PDF file is displayed in the iframe
+    // (caused by sandboxing of the iframe and cross-origin mismatch with
+    // the builtin PDF viewer).
+    // Fall through and load the requested URL.
```

Comment:
I'm not sure to understand the comment here.
If the path only missmatch the `contentIframe.contentWindow.location.pathname`, we simple don't `return`.

If we need a try/catch this is because we have a error (a `undefined` value ?).
Is it because of the sandboxing that we cannot access `location` (or other problematic property) ?

Reply:
> Is it because of the sandboxing that we cannot access location (or other problematic property) ?

Yes. Accessing `.pathname` of `contentIframe.contentWindow.location` results in an exception when the content in the iframe is sandboxed.

Reply:
Ok. Can you update the comment about the exception ?
Once it is done, I think we can merge this PR.

Reply:
Done (while rebasing, without a fixup commit).